api/rest/v2/index.md

```diff
@@ -30,3 +51,84 @@ The base URL for all endpoints below is:
+* All cities in California: “geoId/06&lt;-cityIn”
+* Child stat vars under an SVG: “dc/g/Demographics**<-**memberOf”
+
+The following table describes the new Data Commons REST (v2) relation expressions:
+
+| Symbol      | Represents                                  |
+|-------------|---------------------------------------------|
+| ->          | An “out” arc                                |                      
+| <-          | An “in” arc                                 |                      
+| +           | One or multiple properties chained together, like “containedInPlace+” |
+| *           | Wildcard matches                            |                    
+| #           | A count of nodes                            |                           
+| {p:v}       | Filtering                                   |                           
+| []          | A list of nodes                             |
+{: .doc-table}
+
+Data commons represents real world entities, concepts, and data as nodes. These nodes are connected by a relationship and form a connected knowledge graph. Nodes are connected by edges. One edge represents the relationship between the two nodes, which we also call a “property”. Users can use “property” to query for linked nodes in the knowledge graph. Since the relationships between nodes are directed, we want to represent that with direction, hence using arrow notation, like “->” or “&lt;-”. With this in mind, understand that a property (node relation) has two parts: the direction (represented by arrow notation) and the label. For example “->name”, “&lt;-containedInPlace”.
+
+> **Note**: Properties to query must be represented with relation expressions such as arrow notation. For more information, see the Data Commons REST (v2) API Overview.
+
+## Chaining Properties
+
+In REST (v2) APIIn REST (v2) API, you can represent a chain of the same property with “+”. For example, to find places that are related by containment relationship, we can use “containedInPlace+”. For example, locations in the USA can be represented to find places that are related by containment relationship, we can use “containedInPlace+”. For example, locations in the USA can be represented to find places that are related by containment relationship, we can use “containedInPlace+”. For example:
+
+```
+country/USA<-containedInPlace+
+```
+
+This makes it possible to extend the “property” to more than one node connection.
+
+A filter, represented by “key: value” pairs can be further applied on top of this to limit the target nodes. In the above example, if we only want to get cities in USA, the expression looks like:
+
+```
+country/USA<-containedInPlace+{typeOf: City}
+```
+
+This makes it clear that nodes with “typeOf: City” are queried and are linked to “country/USA” by one or more “containedInPlace” properties.
+
+As an example of property chaining, consider the following diagram:
+
+![alt text](image.png)
```

Comment:
should check in or point to the correct path of image.png.

Can you double check that all the link and text in this page works.

Reply:
I checked in the image and double checked the links.